subsys/usb/usb_c/usbc_prl.c

```diff
@@ -380,7 +380,9 @@ static void prl_tx_set_state(const struct device *dev, const enum usbc_prl_tx_st
 	struct usbc_port_data *data = dev->data;
 	struct protocol_layer_tx_t *prl_tx = data->prl_tx;
 
+#if !defined(CONFIG_TEST)
```

Comment:
Why remove the assert when `CONFIG_TEST` is enabled? The point behind asserts is to alert programmer about conditions that must never be false but do become false due to incorrect code (e.g. invalid API usage).